src/main/java/io/github/syst3ms/skriptparser/lang/ExpressionList.java

```diff
@@ -196,4 +206,5 @@ public void setAndList(boolean isAndList) {
     public Expression<?> getSource() {
         return source != null ? source : this;
     }
+
```

Comment:
Not sure why you're adding all these empty lines at the front and end of classes. You can leave these away in future pull requests.

Reply:
GitHub has a warning stating that the file doesn't end with a new line. Back in the day, programming languages used a new line break to signify the end of the file. Programming languages are more sophisticated now, so they know when the file ends. It's just a standard that everyone uses, and GitHub has a little red icon indicating that the file doesn't end in a new line break, so the SkriptLang team follows suit and always new line breaks at the end of every file.